### PR TITLE
Add list of policies to host info output.

### LINKED ISSUE #155: Add list of policies to host info output
Host info should return list of policies for a host, as that is information users often want.

### PR DESCRIPTION
**Pull request by terjekv**
Adds a comma separated list of policies to the end of `host info`:

```
mreg> host info foo
Name:         foo.example.org
Contact:      admin@example.org
[...]
TXT:          v=spf1 -all
Policies:     fruit, vegetable
```

In other news... We could really generalize and clean up the output commads _a lot_. :(

Fixes #155

### CODE REVIEW

**Review by pederhan — APPROVED** [review]
LGTM

**Comment by oyvindkolbu** [comment]
Not specific to this commit, but I really don't like the timestamp churn.

**Comment by oyvindkolbu** [comment]
> Timestamp churn? I don't follow. :(

In the testsuite. Makes it hard to follow actual changes versus timestamp only changes.

**Comment by terjekv** [comment]
> > Timestamp churn? I don't follow. :(
> 
> In the testsuite. Makes it hard to follow actual changes versus timestamp only changes.

Ah, yeah. There was a brief discussion of removing the timestamps before they hit the test results so new test results don't differ as much. That's something that should be done.
